brother_ql/backends/helpers.py

```diff
@@ -14,14 +14,13 @@
 
 logger = logging.getLogger(__name__)
 
-def discover(backend_identifier='linux_kernel'):
-
+def discover(backend_identifier="pyusb"):
+    if backend_identifier is None:
+        logger.info("Backend for discovery not specified, defaulting to pyusb")
+        backend_identifier = "pyusb"
     be = backend_factory(backend_identifier)
-    list_available_devices = be['list_available_devices']
-    BrotherQLBackend       = be['backend_class']
```

Comment:
Q (blocking): Is there a reason this is replaced?

Reply:
Are you sure the second line has no effect? I like to keep things that ppl. might depend on, even if it seems nonsensical.

Reply:
I've restored this bit so we can get this merged asap.